src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -412,7 +412,23 @@ $ npm install @sentry/browser
 $ npm install @sentry/apm
 ```
 
-#### Sending Traces/Transactions/Spans
+Alternatively, instead of npm packages, you can use our pre-built CDN bundle that combines both `@sentry/browser` and `@sentry/apm`:
+
+```html
+<script src="https://browser.sentry-cdn.com/{% sdk_version sentry.javascript.browser %}/bundle.apm.min.js" crossorigin="anonymous"></script>
+```
+
+*NOTE:* The `Tracing` integration is available then under `Sentry.Integrations.Tracing`.
```

Comment:
This note should go later, after we've talked about what this integration is about and closer to the examples where code needs to be replaced depending on how the SDK is being used (npm vs CDN).